js/layout-javascript/agenda-code.js

```diff
@@ -669,7 +714,11 @@ DynamicList.prototype.attachObservers = function() {
         });
       }
     })
-    .on('click', '.dynamic-list-delete-item', function() {
+    .on('click keydown', '.dynamic-list-delete-item', function(event) {
+      if (event.type === 'keydown' && event.keyCode !== 13 && event.keyCode !== 32) {
```

Comment:
We repeat this so much. Can we simplify this throughout?

```suggestion
      if (!_this.Utils.Event.isExecute(event)) {
```